plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -448,6 +454,11 @@ public static DataSize getParquetSmallFileThreshold(ConnectorSession session)
         return session.getProperty(PARQUET_SMALL_FILE_THRESHOLD, DataSize.class);
     }
 
+    public static boolean isParquetIgnoreStatistics(ConnectorSession session)
```

Comment:
Please add for both Iceberg & Delta at least a test like the following to ensure that the new session property is effective

https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java#L5927-L5961

At best we'd have a table with data files containing broken stats registered from resources.